WordPress/Classes/ViewRelated/Reader/Sorting/ReaderSortingOptionButton.swift

```diff
@@ -0,0 +1,118 @@
+import UIKit
+
+enum ReaderSortingOption: String, CaseIterable {
```

Comment:
Enum that hold possibly sorting options. Currently only two, `date` and `popularity`.